.rubocop.yml

```diff
@@ -6,7 +6,7 @@ AllCops:
   DisplayCopNames: true
 
 Layout/LineLength:
-  Max: 120
+  Max: 250
```

Comment:
Please don't change this file. You should go back and take a look at the methods you are creating and modularize them instead of changing the rules. Remember rubocop serves as a way to standardize code amongst a company and you will have to comply with similar rules when you get a job.